lib/takeaway.rb

```diff
@@ -0,0 +1,41 @@
+require_relative 'order'
+
+class Takeaway
+  attr_reader :order, :menu
+
+  DISHES = {
```

Comment:
I'd like this to be read in from elsewhere if possible. as much as I like the family deal, i'm not sure 1.66 sausages equals a pizza. So your pricing needs work.

(Classes should have 1 reason to change) At the moment if i want to change the dishes i have to change this class